.github/workflows/code-cov.yml

```diff
@@ -5,7 +5,7 @@ name: Generate and archive a Jacoco code coverage report
 on:
   schedule:
     # * is a special character in YAML so you have to quote this string
-    - cron:  '0 0 * * *'
+    - cron:  '30 18 * * *'
```

Comment:
Shall we add a comment saying that this is mid night in LK time. Otherwise this uncommon cron configuration might be confusing for the others.